### DISCUSSION TITLE
Security Onion 2.4 Beta 2 issues

### DISCUSSION THREAD

**Discussion by cedgman** [Security-Onion-Solutions/securityonion#10212] [2.4]
Great work as usual. I do have 2 issue so far. :)

Disabled pcap via the Adminstration -> Configuration -> Pcap does not work.  Neither does editing the minion file directly.

Suricata ignores the threads (default 19) in the minion configuration and always is set to 1.

Thanks!

**Comment by cedgman** [comment]
Also in a distributed deployment, if you add more than one forward/sensor node the dashboards (Dashboards/Kibana) only show data from the first sensor added.

**Reply by weslambert** [reply]
To be clear, does the sensor show as successfully connected in the `Grid` section of SOC? 

Also, is the sensor generating data in `/nsm/pcap`, `/nsm/suricata/`, or `/nsm/zeek/logs/current/`?

**Reply by cedgman** [reply]
Yes.  All 3 sensors are in the Grid with status ok and all 3 are generating data for steno, zeek and suricata.

Only the first sensor I added shows up in any of the dashboards.

**Reply by weslambert** [reply]
If you navigate to `https://$SECURITYONIONMANAGER/kibana/app/fleet/agents`, do you see a healthy status for each Elastic agent reporting for each sensor?

**Reply by cedgman** [reply]
Yes I do.

**Comment by m0duspwnens** [comment]
Thanks for the report. The following issues have been opened:

https://github.com/Security-Onion-Solutions/securityonion/issues/10229
https://github.com/Security-Onion-Solutions/securityonion/issues/10230
https://github.com/Security-Onion-Solutions/securityonion/issues/10231
